#Msg for a cascaded list of links, i.e. an arm
Link_msg[] links
